src/components/core/PureTable.tsx

```diff
@@ -45,7 +46,7 @@ export interface ITableState {
 
 const ROWS_PER_PAGE_IN_RESPONSIVE_TABLE = 9;
 
-class SimpleTable extends React.Component<ITableProps & ILoadingInjectedProps, ITableState> {
+class SimpleTable extends BaseVisualization<ITableProps & ILoadingInjectedProps, ITableState> {
```

Comment:
https://twitter.com/dan_abramov/status/752643494972383232

Reply:
The Table was losing state as it was being destroyed during the loading. It's because the HOC component was rendering LoadingComponent instead of the table. I don't like the inheritance here but I haven't found better solution.